ui/hooks/useTokensToSearch.js

```diff
@@ -59,31 +66,40 @@ export function getRenderableTokenData(
     ) || '';
   const usedIconUrl =
     iconUrl ||
-    (contractMap[toChecksumHexAddress(address)] &&
-      `images/contract/${contractMap[toChecksumHexAddress(address)].logo}`);
+    (tokenList[tokenAddress] &&
+      `images/contract/${tokenList[tokenAddress].iconUrl}`);
```

Comment:
If tokenList is static won't this still be `logo`?

Reply:
No, in TokenListController we are reassigning `logo` to `iconUrl`.
https://github.com/MetaMask/controllers/blob/c7d7480339b9dc62bdd2a2b8cada49a96d62b01e/src/assets/TokenListController.ts#L249